BillTechPaymentsUpdater.php

```diff
@@ -106,7 +106,7 @@ function updatePayments()
 				continue;
 			}
 
-			$ids = $DB->GetCol("SELECT id FROM billtech_payments WHERE token=?", array($payment->token));
+			$ids = $DB->GetCol("SELECT id FROM billtech_payments WHERE token=? or reference_number=?", array($payment->token, $payment->reference_number));
```

Comment:
Czemu tak?

Reply:
Obecnie jak się pobiera dane od początku (np. dla debug=true) w momencie migracji wtyczki, to tworzą się duplikaty płatności (w starej wtyczce wpłaty tymczasowe nie miały tokenu). Ewentualnie można filtrować po samym reference_number.

Reply:
Pytanie czy to nie spowoduje problemu w nowej integracji gdzie może być jedna płatność za kilka faktur - wtedy wszystkie mają ten sam reference_number.

Reply:
Racja, poprawiam.